footprints/main/models.py

```diff
@@ -327,6 +327,7 @@ def __unicode__(self):
 class DigitalObject(models.Model):
     name = models.CharField(max_length=500)
     file = models.FileField(upload_to="%Y/%m/%d/")
+    url = models.CharField(max_length=256, blank=True)
```

Comment:
I worry that 256 chars is too little for a URL field.

As a general rule, I avoid using `CharField`s altogether and just stick to `TextField`'s that don't have limits. Character limits on fields mean that anywhere that field might be modified, we now have to add a check/UI to make sure someone doesn't try to go over the limit or they'll get a 500.